jdk_17_maven/cs/rest/signal-server/api-doc/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>TextSecureServer</artifactId>
     <groupId>org.whispersystems.textsecure</groupId>
-    <version>JGITVER</version>
+    <version>10.3.0</version>
```

Comment:
can leave JGITVER, ie, original value. but remove (comment out) the plugin

Reply:
actually, can use numbers, but put a `-SNAPSHOT` at the end, ie `10.3.0-SNAPSHOT`. 

do not remove stuff, butcomment out, ie: 

```
<!-- <version>JGITVER</version> -->
<version>10.3.0-SNAPSHOT</version>
```